.github/workflows/depviz.yml

```diff
@@ -0,0 +1,15 @@
+on:
+  workflow_dispatch:
+  push:
+
+jobs:
+  generate:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: Doozers/depviz-action@e68e52e8761f640c9dd3bb3dd5859c7f4d29bbe5
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
I think this is unnecessary, there is a limited token by default that could be enough, can you check?

Reply:
Yes 'll look at it, but this secret is by default.